src/SUFST/src/can_unpack.c

```diff
@@ -10,15 +10,19 @@
 #define QUEUE_RX_THREAD_PRIORITY             10
 #define QUEUE_RX_THREAD_STACK_SIZE           1024
 #define QUEUE_RX_THREAD_PREEMPTION_THRESHOLD 10
+#define STATS_TIMER_TICKS                    100
+#define STATS_TIMER_SECONDS                 STATS_TIMER_TICKS/100
```

Comment:
Use `TX_TIMER_TICKS_PER_SECOND` instead of the `100`, in case the tick rate is ever changed in future.